CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+### 2.21.1
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.9](https://fluentbit.io/announcements/v1.8.9/)
+* Amazon CloudWatch Logs for Fluent Bit 1.6.3
+* Amazon Kinesis Streams for Fluent Bit 1.8.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.21.0` this release adds:
```

Comment:
Kept "Compared to..." since we have an actual bug fix from @hossain-rayhan

Reply:
yea but the compared too isn't true for the other ones. So Its still a lie. I think we should change the language for the 2 fixes we keep adding in.

Reply:
Oh okay. I see. Will fix.